### PR TITLE
Update Yahoo_Japan.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by renyouguo** [comment]
2016-05-14 17_43_37- - yahoo japan

File: src/chrome/content/rules/Yahoo_Japan.xml [hunk 1]
```diff
@@ -176,7 +175,8 @@
 	<target host="b7.yahoo.co.jp" />
 	<target host="b8.yahoo.co.jp" />
 	<target host="b9.yahoo.co.jp" />
-	<target host="b10.yahoo.co.jp" />
+	<!--	Mismathc:
```
Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: typo

Comment (renyouguo) [inline src/chrome/content/rules/Yahoo_Japan.xml]: @J0WI What does "typo" mean?

Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: a typo is a small misspelling

File: src/chrome/content/rules/Yahoo_Japan.xml [hunk 2]
```diff
@@ -180,23 +175,30 @@
...
 
 	<target host="forms.business.yahoo.co.jp" />
 	<target host="ols.business.yahoo.co.jp" />
 	<target host="yadui.business.yahoo.co.jp" />
 
 	<target host="biz-form.ec.yahoo.co.jp" />
-	<target host="help.yahoo.co.jp" />
-	<target host="docs.id.yahoo.co.jp" />
+	<!--target host="help.yahoo.co.jp" /-->
+	<!--target host="docs.id.yahoo.co.jp" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Yahoo_Japan.xml]: non existent hosts can be removed.

**Comment by J0WI** [comment]
Thanks!

**Comment by luixxiul** [comment]
I'm pushing a PR to include ``auctions.yahoo.co.jp``. Hope it will be merged soon.

**Pull request merged**